src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -111,6 +111,10 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('security_rememberme.xml');
 
         if ($this->authenticatorManagerEnabled = $config['enable_authenticator_manager']) {
+            if ($config['always_authenticate_before_granting']) {
+                throw new InvalidConfigurationException('The security options "always_authenticate_before_granting" cannot be used when "enable_authenticator_manager" is set to true.');
```

Comment:
Minor plural->singular note and I think we should point to the solution: Not using the new system (for now). Now I've flipped the review and I'm open to any improvement on my suggestion 😄 

```suggestion
                throw new InvalidConfigurationException('The security option "always_authenticate_before_granting" cannot be used when "enable_authenticator_manager" is set to true. If you rely on this behavior, use "enable_authenticator_manager: false".');
```